### DISCUSSION TITLE
How to position the machine using a dead wheel? (Whitch is a Localizer of Dead Wheel)

### DISCUSSION THREAD

**Discussion by 6-BennyLi-9** [FIRST-Tech-Challenge/FtcRobotController#1027] [Q&A] [answered]
We all know that ```road runner``` help us do that in many cases by ```TwoDeadWheelLocalizer``` and ```ThreeDeadWheelLocalizer``` in 1.0. And we also know that how to tune it and use it.

But now I want to know how it works in basic. I tried to use Complex and Some other Math knowledge to do this. And I got my result, 
 you can see it in my repo ```FTC-Robot-Integration-Client```. But however, it doesn't look that well.

By the way, I'm a Chinese so I can't open [YouTube](http://cms.youtube.com/) links.

**Comment by cmacfarl** [comment]
> I can't open [YouTube](http://cms.youtube.com/) links, if you want to recommend me a video about it, please download it and drop it in GitHub

Please do not do this.  Downloading video is a violation of YouTube's terms of service.   https://www.youtube.com/static?template=terms

I don't have the math background to be able to answer your question, but I presume you want to ask your favorite search engine to return results for "kinematics" or "ftc robot kinematics".

**Reply by 6-BennyLi-9** [reply]
Sorry about that I don't know anything about youtube(I only know there is a video website called youtube).

**Reply by 6-BennyLi-9** [reply, marked as answer]
Thank you very much because I found the website that I really want: [link](https://rr.brott.dev/papers/Mobile_Robot_Kinematics_for_FTC.html)
